plugins/node/opentelemetry-instrumentation-express/test/hooks.test.ts

```diff
@@ -169,4 +169,140 @@ describe('ExpressInstrumentation hooks', () => {
       );
     });
   });
+
+  describe('span attributes hooks', () => {
+    let server: http.Server;
+    let port: number;
+    let rootSpan: Span;
+
+    beforeEach(async () => {
+      rootSpan = tracer.startSpan('rootSpan');
+
+      const httpServer = await serverWithMiddleware(tracer, rootSpan, app => {
+        app.get('*', (req, res) => {
+          res.send('ok');
+        });
+      });
+      server = httpServer.server;
+      port = httpServer.port;
+    });
+
+    afterEach(() => {
+      server.close();
+    });
+
+    it('should set span attributes', async () => {
+      instrumentation.setConfig({
+        spanAttributesHook: ({ request, route, layerType }, baseAttributes) => {
+          if (layerType) {
+            return {
+              route,
+            };
+          }
+
+          if (route === '*') {
+            return {
+              method: request.method,
+              url: request.url,
```

Comment:
It's completely valid to use any string for the attribute name, but in OpenTelemetry those specific attributes are defined in semantic conventions specification and are called `http.method`(`SemanticAttributes.HTTP_METHOD`) and `http.url`(`SemanticAttributes.HTTP_URL`).

I think it will be more appropriate to use semantic names here because current values promote non-spec compliant usage.

Reply:
Sure. I used those values here, same as they are for the name hook. I'll open another PR to change them in there as well.

Reply:
Sure. I just reused this as it is for the name hook. I'll change it for the attributes hook in this PR and open another one for name.